internal/services/systemcentervirtualmachinemanager/system_center_virtual_machine_manager_virtual_network_resource_test.go

```diff
@@ -0,0 +1,229 @@
+	"github.com/hashicorp/terraform-provider-azurerm/internal/clients"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/tf/pluginsdk"
+	"github.com/hashicorp/terraform-provider-azurerm/utils"
+)
+
+type SystemCenterVirtualMachineManagerVirtualNetworkResource struct{}
+
+func TestAccSystemCenterVirtualMachineManagerVirtualNetworkSequential(t *testing.T) {
+	// NOTE: this is a combined test rather than separate split out tests because only one System Center Virtual Machine Manager Server can be onboarded at a time on a given Custom Location
+
+	if os.Getenv("ARM_TEST_CUSTOM_LOCATION_ID") == "" || os.Getenv("ARM_TEST_FQDN") == "" || os.Getenv("ARM_TEST_USERNAME") == "" || os.Getenv("ARM_TEST_PASSWORD") == "" {
+		t.Skip("Skipping as one of `ARM_TEST_CUSTOM_LOCATION_ID`, `ARM_TEST_FQDN`, `ARM_TEST_USERNAME`, `ARM_TEST_PASSWORD` was not specified")
+	}
+
+	acceptance.RunTestsInSequence(t, map[string]map[string]func(t *testing.T){
+		"scvmmVirtualNetwork": {
+			"basic":          testAccSystemCenterVirtualMachineManagerVirtualNetwork_basic,
+			"requiresImport": testAccSystemCenterVirtualMachineManagerVirtualNetwork_requiresImport,
+			"complete":       testAccSystemCenterVirtualMachineManagerVirtualNetwork_complete,
+			"update":         testAccSystemCenterVirtualMachineManagerVirtualNetwork_update,
+		},
+	})
+}
+
+func testAccSystemCenterVirtualMachineManagerVirtualNetwork_basic(t *testing.T) {
+	data := acceptance.BuildTestData(t, "azurerm_system_center_virtual_machine_manager_virtual_network", "test")
+	r := SystemCenterVirtualMachineManagerVirtualNetworkResource{}
+
+	data.ResourceSequentialTest(t, r, []acceptance.TestStep{
+		{
+			Config: r.basic(data),
+			Check: acceptance.ComposeTestCheckFunc(
+				check.That(data.ResourceName).ExistsInAzure(r),
+			),
+		},
+		data.ImportStep(),
+	})
+}
+
+func testAccSystemCenterVirtualMachineManagerVirtualNetwork_requiresImport(t *testing.T) {
+	data := acceptance.BuildTestData(t, "azurerm_system_center_virtual_machine_manager_virtual_network", "test")
+	r := SystemCenterVirtualMachineManagerVirtualNetworkResource{}
+
+	data.ResourceSequentialTest(t, r, []acceptance.TestStep{
+		{
+			Config: r.basic(data),
+			Check: acceptance.ComposeTestCheckFunc(
+				check.That(data.ResourceName).ExistsInAzure(r),
+			),
+		},
+		data.RequiresImportErrorStep(r.requiresImport),
+	})
+}
+
+func testAccSystemCenterVirtualMachineManagerVirtualNetwork_complete(t *testing.T) {
+	data := acceptance.BuildTestData(t, "azurerm_system_center_virtual_machine_manager_virtual_network", "test")
+	r := SystemCenterVirtualMachineManagerVirtualNetworkResource{}
+
+	data.ResourceSequentialTest(t, r, []acceptance.TestStep{
+		{
+			Config: r.complete(data),
+			Check: acceptance.ComposeTestCheckFunc(
+				check.That(data.ResourceName).ExistsInAzure(r),
+			),
+		},
+		data.ImportStep(),
+	})
+}
+
+func testAccSystemCenterVirtualMachineManagerVirtualNetwork_update(t *testing.T) {
```

Comment:
shall we use the order of basic->complete->update->basic?

Reply:
Updated